qiskit/visualization/latex.py

```diff
@@ -257,14 +255,18 @@ def _initialize_latex_array(self):
                             virt_reg = next(
                                 reg for reg in self.layout.get_registers() if virt_bit in reg
                             )
-                            label = "\\lstick{{ {{{}}}_{{{}}}\\mapsto{{{}}} : ".format(
+                            label = "\\nghost{Register: " + virt_reg.name + "} & "
+                            label += "\\lstick{{ {{{}}}_{{{}}}\\mapsto{{{}}} : ".format(
                                 virt_reg.name, virt_reg[:].index(virt_bit), bit_location["index"]
                             )
                         except StopIteration:
-                            label = "\\lstick{{ {{{}}} : ".format(bit_location["index"])
+                            label = "\\nghost{Register: " + bit_location["index"] + "} & "
+                            label += "\\lstick{{ {{{}}} : ".format(bit_location["index"])
                     else:
-                        label = "\\lstick{{ {{{}}} : ".format(bit_location["index"])
+                        label = "\\nghost{Register: " + bit_location["index"] + "} & "
+                        label += "\\lstick{{ {{{}}} : ".format(bit_location["index"])
                 if self.initial_state:
+                    #label += "\\ket{{0}}"
                     label += "\\ket{{0}}"
                 label += " }"
                 self._latex[i][0] = label
```

Comment:
Again, this is just a small suggestion. It is up to you if you want to put it in. Since the ```label``` (of qubit) is assigned to ```self._latex[i][0]``` in line 272, instead of adding ```\nghost``` at various places, you could simply have the  parameter of ```\lstick``` in a variable and then just above line 272 you could add ```\nghost``` in front of ```label```.